SDL/SDLMain.cpp

```diff
@@ -176,6 +177,13 @@ void LaunchEmail(const char *email_address) {
 }
 
 std::string System_GetProperty(SystemProperty prop) {
+#ifdef __linux__
```

Comment:
This check should be done in the case SYSPROP_LANGREGION, not out here, and preferably only once during execution

Reply:
note that you can declare variables in a switch case, you just have to wrap the entire case in { }.